Source/WebCore/Modules/model-element/ModelPlayer.cpp

```diff
@@ -46,6 +51,11 @@ bool ModelPlayer::supportsDragging()
     return true;
 }
 
+bool ModelPlayer::supportsTransform(TransformationMatrix)
+{
+    return true;
```

Comment:
Would false be a better default value here?

Reply:
Makes sense! I'll make it false by default.